### PR TITLE
Add dockerfile

### PR DESCRIPTION
**Pull request by bgpat**
## WHY

To dockerlize https://github.com/golang/gddo.

## WHAT

Add `Dockerfile`.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM golang:1.6-alpine
```
Comment (dtan4) [inline Dockerfile]: Current latest version is 1.8 😎

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+RUN apk --update --no-cache add git && \
+	go get github.com/golang/gddo/gddo-server && \
+	go install github.com/golang/gddo/gddo-server
```
Comment (dtan4) [inline Dockerfile]: Uninstall `git` here may reduces the total image size.

Comment (bgpat) [inline Dockerfile]: Remove git package. Please review :bow:

File: Dockerfile (hunk 1 above)
Comment (bgpat) [inline Dockerfile]: Official document say: `Install Go1.6`.
https://github.com/golang/gddo/wiki/Development-Environment-Setup

Comment (dtan4) [inline Dockerfile]: Okay, please comment it above this line.

Comment (bgpat) [inline Dockerfile]: Add comment.

**Comment by dtan4** [comment]
LGTM

**Comment by koudaiii** [comment]
LGTM 👍

**Pull request merged**